libraries/AP_HAL_ChibiOS/hwdef/LumenierLUXF765-NDAA/README.md

```diff
@@ -0,0 +1,370 @@
+
+
+### GPS port
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>+5V</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>TX3</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX3</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>I2C3 SCL</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>I2C3 SDA</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### HD VTX port
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>+10V</td>
+   <td>+10V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>TX4</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>RX4</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>RX7</td>
+   <td>+3V3</td>
+   </tr>
+   </tbody>
+   </table>
+   
+### Receiver Port
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>+5V</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX7</td>
+   <td>+3.3</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>TX7</td>
+   <td>+3.3V</td>
+   </tr>
+   </tbody>
+   </table>
+   
+### CANbus Port
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>+5V</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CAN_H</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>CAN_L</td>
+   <td>+5V</td>
+   </tr>
+   </tbody>
+   </table>
+
+## PWM Output
+
+The Lumenier LUX F765 NDAA supports 12 PWM outputs. 
+The 8 main PWM outputs are labeled M1 through M8.
+The 4 auxiliary outputs are labeled S1 through S4. 
+
+The 8 main PWM outputs are in 2 groups:
+
+ - PWM 1 through 4 (M1 - M4) are in TIM2
+ - PWM 5 through 8 (M5 - M8) are in TIM4
+
+The 4 auxiliary PWM outputs are in 1 group:
+ - PWM 9 through 12 (S1 - S4) are in TIM1
+
+## Analog inputs
+
+The LUX F765 NDAA has 2 analog inputs:
+
+ - PC2 -> Battery Current
+ - PC3 -> Battery Voltage
+
+## Loading Firmware
+
+The board comes pre-installed with Betaflight firmware when purchased from GetFPV. 
+The board can be easily transitioned from Betaflight to Ardupilot in a matter of minutes. 
+Please see the [product page](https://www.getfpv.com/lumenier-lux-f765-flight-controller-ndaa.html)
+for firmware binaries and instructions on how to transition from Betaflight to Ardupilot.
+
```

Comment:
-needs section on where to find firmware
-needs sections for RC Input,Compass, OSD, Battery Monitor, and Camera Switch

See Wiki: https://ardupilot.org/dev/docs/readme_file.html  (just merged give it a few hours to appear)